addons/l10n_lu/data/account_tax_report_line.xml

```diff
@@ -463,6 +464,7 @@
       <field name="sequence">6</field>
       <field name="parent_id" ref="account_tax_report_line_2a_breakdown_taxable_turnover_base"/>
       <field name="country_id" ref="base.lu"/>
+      <field name="code">LUTAX_701</field>
```

Comment:
Not used?

Reply:
fixed the missing `account_id`s and a `plus` instead of `minus` too